src/components/rules-creation-page/SyncRulesWizard.jsx

```diff
@@ -36,6 +37,14 @@ class SyncRulesWizard extends React.Component {
             description: undefined,
             help: undefined,
         },
+        {
+            key: "include-exclude-selection",
+            label: i18n.t("Include Exclude Selection"),
+            component: IncludeExcludeStep,
+            validationKeys: ["includeExclude"],
```

Comment:
[minor] Why don't we use metadataIncludeExclude for the validation key? Here we only have Metadata but when porting to data-Sync could be confusing.

Reply:
Done!